test/functional/page_objects/dashboard_page.ts

```diff
@@ -302,10 +302,19 @@ export class DashboardPageObject extends FtrService {
     });
     await this.testSubjects.click('dashboardViewOnlyMode');
     if (accept) {
-      const confirmation = await this.testSubjects.exists('confirmModalTitleText');
-      if (confirmation) {
-        await this.common.clickConfirmOnModal();
-      }
+      await this.common.clickConfirmOnModal();
```

Comment:
The check for `confirmModalTitleText` already happens as part of `clickConfirmOnModal` so no need to do this twice